uk.co.compucorp.civicrm.hrcore/backstop_data/scenarios/ssp-la-my-report.json

```diff
@@ -1,18 +1,8 @@
 {
   "scenarios": [{
```

Comment:
I know this is not part of your PR, but if you could do the same thing for the bracket it would be great.